website/source/docs/job-specification/service.html.md

```diff
@@ -89,6 +89,12 @@ does not automatically enable service discovery.
     - `${GROUP}` - the name of the group
     - `${TASK}` - the name of the task
     - `${BASE}` - shorthand for `${JOB}-${GROUP}-${TASK}`
+    
+    Any environment variables in the name are ignored during validation, because some environment variables can only
```

Comment:
How about:

Validation of the name occurs in two parts. When the job is registered, an initial validation pass checks that the service name adheres to [RFC-1123 §2.1](https://tools.ietf.org/html/rfc1123#section-2) and the length limit, excluding any variables requiring interpolation. Once the client receives the service and all interpretable values are available, the service name will be interpolated and revalidated. This can cause certain service names to pass validation at submit time but fail at runtime.